### ISSUE TITLE
Different speed of the various Lizmap versions

### ISSUE THREAD

**Issue by guenterw** [3liz/lizmap-web-client#4379]
I have now activated http2 but this does not bring any change. The measurements with lighthouse fluctuate a lot and give a completely different picture than the normal test in the browser.
So I preferred to stop the speed with the watch. Several times, each time clearing the browser cache.
It is interesting that there are big differences between the PC (older i5 Win10), the notebook (newer i7 Win10) and mobile devices (Samsung tablet).
In each case from the start of the project (Bauleitplanung),
![image](https://github.com/3liz/lizmap-web-client/assets/8373176/79874c96-2bd8-4f68-96cb-8f004cf68bc2)

immediately zoomed to a maximum scale of 1:250 (PC and notebook with mouse scroll wheel):

https://cx21-6.webgis.biz/ Lizmap 3.6.11:  PC 10 sec., NB 5 sec. Mobile < 5 sec.
https://cx21-7.webgis.biz/ Lizmap 3.7.6:    PC 30 sec., NB 16 sec., Mobile 10-15 sec.
https://cx21-8.webgis.biz/ Lizmap 3.8.0 Alpha.1  PC 54 sec., NB 26 sec., Mobile 10-15 sec.

How can we explain these differences?
And version 3.6 is always significantly faster in all tests

With versions 3.7 and 3.8 you sit in front of a white screen for a long time (especially on the PC)
![image](https://github.com/3liz/lizmap-web-client/assets/8373176/ba5f558a-8c3e-49d0-a8ab-729d40bb94dd)

_Originally posted by @guenterw in https://github.com/3liz/lizmap-web-client/issues/4367#issuecomment-2071965304_

**Comment by Gustry** [comment]
Fix is already ready, thanks to @rldhont 

You should have a look to URL used in your GetMap requests ;-)

**Comment by rldhont** [comment]
@guenterw I invite you to test https://github.com/3liz/lizmap-web-client/pull/4380

**Comment by Gustry** [comment]
This is issue is linked how your layers are fetched : either direct to QGIS server, or using an external server

**Comment by Gustry** [comment]
It should be a duplicate of #4318

**Comment by guenterw** [comment]
Question: what is the best way to test/install the master version?
I copied the lizmap directory and then did the usual steps of an update (configurator.php/installer.php/...)
The backend works and the alpha.2 is also displayed but no map is loaded.
![image](https://github.com/3liz/lizmap-web-client/assets/8373176/3432beae-ed72-4df6-8d23-a51abce7734e)
No error message in the log

**Comment by Gustry** [comment]
> Question: what is the best way to test/install the master version?

Now that the PR is merged, you can wait for the latest ZIP available https://packages.3liz.org/

**Comment by Gustry** [comment]
>  I copied the lizmap directory and then did the usual steps of an update (configurator.php/installer.php/...)

You must build the JS and PHP : https://github.com/3liz/lizmap-web-client/blob/master/INSTALL.md#get-the-source

**Comment by rldhont** [comment]
@guenterw the PR has been merged, you can find the fix in unstable 3.8.0-alpha.2 here https://packages.3liz.org/pub/lizmap/unstable/3.8/

**Comment by guenterw** [comment]
perfect! Many thanks for the quick implementation and for the Alpha.2
This is now quite fast, especially for the small test server.

I have also created a test project "ALKIS aus DB" (https://cx21-8.webgis.biz) without WMS services, but only with data from the PG-DB.
Here, too, I find the behavior of the “rotating circles” is now better than in 3.7 (https://cx21-7.webgis.biz). Or the 3.8 is simply faster.

However, I also noticed another error during testing. Layers that are combined into a group are not automatically displayed as active, even though this is defined. This is why the “ALKIS-Daten” group must first be activated in the example. I will write a new issue for this. In version 3.6 (https://cx21-6.webgis.biz) it works (it is always the same project, only the version has been adjusted in the plugin)
